securing_apps/topics/token-exchange/token-exchange.adoc

```diff
@@ -313,24 +326,23 @@ and or refresh token depending on the `requested_token_type` parameter value.  Y
 user session will remain active until it times out or until you call the logout endpoint of the realm passing this
 new access token.
 
-These types of changes required a configured identity provider in the admin console.
+These types of changes required a configured identity provider in the Admin Console.
 
 NOTE:  SAML identity providers are not supported at this time.  Twitter tokens cannot be exchanged either.
 
+==== Granting permission for the exchange
 
-==== Granting Permission for the Exchange
-
-Before external token exchanges can be done, you must grant permission for the calling client to make the exchange.  This
+Before external token exchanges can be done, you grant permission for the calling client to make the exchange.  This
 permission is granted in the same manner as <<_grant_permission_external_exchange, internal to external permission is granted>>.
 
 If you also provide an `audience` parameter whose value points to a different client other than the calling one, you
 must also grant the calling client permission to exchange to the target client specific in the `audience` parameter.  How
 to do this is <<_client_to_client_permission, discussed earlier>> in this section.
 
-==== Making the Request
+==== Making the request
 
-The `subject_token_type` must either be `urn:ietf:params:oauth:token-type:access_token` or `urn:ietf:params:oauth:token-type:jwt`.
-If the type is `urn:ietf:params:oauth:token-type:access_token` you must specify the `subject_issuer` parameter and it must be the
+the `subject_token_type` must either be `urn:ietf:params:oauth:token-type:access_token` or `urn:ietf:params:oauth:token-type:jwt`.
```

Comment:
For me the previous text was fine. The `the` should be `The`? And `must` really applies there.

Reply:
Okay, thanks.